modules/nixos/buildbot.nix

```diff
@@ -14,6 +14,15 @@
     "http://localhost:8011/metrics"
   ];
 
+  systemd.services.buildbot-master.serviceConfig = {
+    Restart = "on-failure";
+    RestartSec = "30s";
+  };
+  systemd.services.buildbot-worker.serviceConfig = {
+    Restart = "on-failure";
```

Comment:
Afaik the worker shouldn't be affected by this. The github fetching logic should be only loaded by the buildbot master process.

Reply:
Does the worker have any sort of timeout if it can't connect to the master or will it idle indefinitely? I have seen it fail but can't remember the specific error.

Reply:
I'll drop it for the worker, easy enough to add it back if necessary.

https://github.com/nix-community/infra/pull/1248

Reply:
Afaik it reconnects eventually.